src/fixup.js

```diff
@@ -194,7 +198,18 @@
           button.innerText = (isOpen) ? '\u25BE collapse' : '\u25B4 expand';
         }
       }
+
       document.body.appendChild(node);
+      button.focus();
+      window.onkeydown = function (event) {
+        if (event.keyCode == 27 && !node.classList.contains("outdated-collapsed")) button.click();
```

Comment:
nit:  "===". Also, to avoid potential bugs later, add the "{" and "}" around the if; 

Might help to also do the following... for legibility:

```JS
var ESCAPEKEY = 27; // top of this file

if (event.keyCode === ESCAPEKEY && ...)
```